src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -95,8 +95,15 @@ public static void merge(List<RepoConfiguration> repoConfigs, List<AuthorConfigu
             RepoConfiguration matchingRepoConfig = getMatchingRepoConfig(repoConfigs, authorConfig);
 
             if (matchingRepoConfig == null) {
-                logger.warning(String.format(
-                        "Repository %s is not found in repo-config.csv.", authorConfig.getLocation()));
+                if (authorConfig.isUsingDefaultBranch()) {
+                    logger.warning(String.format(
+                            "Repository %s is not found in repo-config.csv.",
+                            authorConfig.getLocation(), authorConfig.getBranch()));
```

Comment:
perhaps, let's do something like this:
```
String branchInfo = authorConfig.isDefaultBranch() 
        ? String.format(" (branch %s)",0 authorConfig.getBranch()) 
        : "";
logger.warning(String.format(
        "Repository %s %s is not found in repo-config.csv.",
        authorConfig.getLocation(), branchInfo));
```
Generally try to use lesser long strings as it affects the size of compiled output.

Reply:
Fixed